### ISSUE TITLE
If the dataset carla_dataset lacks training data?

### ISSUE THREAD

**Issue by onetrance** [idrl-lab/Full-coverage-camouflage-adversarial-attack#8]
Excuse me, there is only one data1.png in train_new in carla_dataset, but there are a large number of labels in train_label_new. How to download the data sets corresponding to train_label_new in train_new?

**Comment by onetrance** [comment]
Can the author or a friend guide me? Grateful！Thanks！！！

**Comment by MTzhong** [comment]
you can download the dataset in  https://drive.google.com/drive/folders/1vspvRxnZ3shOV4kM5ELcO9-xztapBThS

**Comment by MTzhong** [comment]
in the paper "To compare with previous works, we use the same datasets provided by (Wang et al.2021a) 
directly"

**Comment by lxch1987** [comment]
How many epochs is needed to train these data, please?

**Comment by Francisapzii** [comment]
> you can download the dataset in https://drive.google.com/drive/folders/1vspvRxnZ3shOV4kM5ELcO9-xztapBThS
Tnanks.
From this link I download the zip file. But can't found “train_new”，just “train”.
So where to get “train_new”?

**Comment by zhanz5** [comment]
> > you can download the dataset in https://drive.google.com/drive/folders/1vspvRxnZ3shOV4kM5ELcO9-xztapBThS
> > Tnanks.
> > From this link I download the zip file. But can't found “train_new”，just “train”.
> > So where to get “train_new”?

hello, have you solved it?  where to get “train_new”?
